Strapi-maintained plugins descriptions (#833)
strapi/documentation: docs/user-docs/latest/plugins/strapi-plugins.md

> | Content-type Builder | <ul><li>Addition of a new setting at the Content-type level, to allow or not localisation/translation of the Content-type (see [Creating a new content-type](/user-docs/latest/content-types-builder/creating-new-content-type.md#creating-a-new-content-type)).</li> <li>Addition of a new setting at field level, to allow or not localisation/translation of the Content-type (see [Configuring fields for content-types](/user-docs/latest/content-types-builder/configuring-fields-content-type.md#regular-fields)).</li></ul> |

Reviewer: ```suggestion
| Content-type Builder | <ul><li>Addition of a new setting at the Content-type level, to allow or not localization/translation of the content-type (see [Creating a new content-type](/user-docs/latest/content-types-builder/creating-new-content-type.md#creating-a-new-content-type)).</li> <li>Addition of a new setting at field level, to allow or not localization/translation of the content-type (see [Configuring fields for content-types](/user-docs/latest/content-types-builder/configuring-fields-content-type.md#regular-fields)).</li></ul> |
```

The Email plugin allows users to send email from the server or from external providers such as Sendgrid. The Email plugin is not configurable in the admin panel, however users can test email delivery if it has been setup by an administrator. More information about the email plugin is available in the [Developer Documentation](/developer-docs/latest/plugins/email.md).

| Section impacted | Options and settings |
|------------------|----------------------|
| Settings         | <ul><li>Addition of "Email plugin" setting section, which contains a "Configuration" sub-section. In the Configuration section, only the email address field under "Test email delivery" is modifiable by users. A **send test email** button sends a test email.</li> <li>Addition of "Email" to the permissions for authenticated and public users. In the Email section the ability to send emails via the API can be enabled or disabled. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Users and Permissions > Roles* </li></ul>|

## Additional plugins

### <img width="28" src="../assets/plugins/Documentation-swagger.png"> Documentation

>  The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation).  The Documentation plugin allows you to:

Reviewer 2: ```suggestion
 The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation). The Documentation plugin allows to:
```

Author: reworded to be grammatically correct.

>  The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation).  The Documentation plugin allows you to:

Reviewer 2: ```suggestion
 The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation). The Documentation plugin allows to:
```

- open the API documentation,
- regenerate the documentation,
- restrict access to the documentation endpoint.

> The Documentation plugin affects multiple parts of the admin panel. The table below lists all the additional options and settings that are added to a Strapi application once the plugin has been installed.

Reviewer 2: I just noticed that this introductory sentence before the table is not used for all plugin docs, maybe it would be better to add it everywhere there's a table (as always, for more clarity & consistency) 🤓

Author: done

> The Documentation plugin affects multiple parts of the admin panel. The table below lists all the additional options and settings that are added to a Strapi application once the plugin has been installed.

Reviewer 2: I just noticed that this introductory sentence before the table is not used for all plugin docs, maybe it would be better to add it everywhere there's a table (as always, for more clarity & consistency) 🤓

Author: duplicate comment

| Section impacted    | Options and settings         |
|------------------|---------------------------------------------------------------------------------------------------------|
| Documentation    | <ul>Addition of a new Documentation option in the main navigation under the plugins heading, which contains links to open and refresh the documentation.   </ul>        |

> | Settings     | <ul><li>Addition of a "Documentation plugin" setting section, which controls whether the documentation endpoint is private or not. <br> 👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > documentation plugin* </li><br>  <li> Activation of role based access control for accessing, updating, deleting, and regenerating the documentation. Administrators can authorize different access levels to different types of users in the *Plugins* tab and the *Settings* tab. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Administration Panel > Roles*  <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted"/> </li></ul>|

Reviewer 2: ```suggestion
| Settings     | <ul><li>Addition of a "Documentation plugin" setting section, which controls whether the documentation endpoint is private or not. <br> 👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Documentation plugin* </li><br>  <li> Activation of role based access control for accessing, updating, deleting, and regenerating the documentation. Administrators can authorize different access levels to different types of users in the *Plugins* tab and the *Settings* tab. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Administration Panel > Roles*  <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted"/> </li></ul>|
```

Author: fixed

> | Settings     | <ul><li>Addition of a "Documentation plugin" setting section, which controls whether the documentation endpoint is private or not. <br> 👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > documentation plugin* </li><br>  <li> Activation of role based access control for accessing, updating, deleting, and regenerating the documentation. Administrators can authorize different access levels to different types of users in the *Plugins* tab and the *Settings* tab. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Administration Panel > Roles*  <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted"/> </li></ul>|

Reviewer 2: ```suggestion
| Settings     | <ul><li>Addition of a "Documentation plugin" setting section, which controls whether the documentation endpoint is private or not. <br> 👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Documentation plugin* </li><br>  <li> Activation of role based access control for accessing, updating, deleting, and regenerating the documentation. Administrators can authorize different access levels to different types of users in the *Plugins* tab and the *Settings* tab. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Administration Panel > Roles*  <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted"/> </li></ul>|
```

### <img width="28" src="../assets/plugins/Gatsby_Monogram.png"> Gatsby preview

The Gatsby preview plugin allows applications with Gatsby Cloud accounts to preview the front end. The Strapi Gatsby preview plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-gatsby-preview).

| Section impacted    | Options and settings         |
|------------|-----------------|

> | Settings     |  Addition of the Gatsby preview plugin section in the Settings sub navigation. In the plugin settings the user can: <ul><li> enable or disable Collection Types and Single Types, </li> <li>add the Gatsby Content Sync URL</li> </ul> |

Reviewer 2: ```suggestion
| Settings     |  Addition of a "Gatsby preview plugin" setting section, enables/disables collection types and single types, and allows to add the Gatsby Content Sync URL. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Gatsby preview plugin* |
```

Can you check that the setting section name is "Gatsby preview plugin" written like so? :)

Author: <img width="167" alt="Screen Shot 2022-05-06 at 16 47 40" src="https://user-images.githubusercontent.com/97027841/167157054-b0fe2220-6caf-4de0-803c-aa5c63472e55.png">

Author: I don't think we have anything in the docs in caps case, correct? I have left this as-is.

Author: reworded here to be grammatically correct.

> | Settings     |  Addition of the Gatsby preview plugin section in the Settings sub navigation. In the plugin settings the user can: <ul><li> enable or disable Collection Types and Single Types, </li> <li>add the Gatsby Content Sync URL</li> </ul> |

Reviewer 2: ```suggestion
| Settings     |  Addition of a "Gatsby preview plugin" setting section, enables/disables collection types and single types, and allows to add the Gatsby Content Sync URL. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Gatsby preview plugin* |
```

Can you check that the setting section name is "Gatsby preview plugin" written like so? :)

> | Content Manager     | Addition of the **open Gatsby preview** button in the right-side navigation                  |

Reviewer 2: ```suggestion
| Content Manager     | Addition of the **open Gatsby preview** button in the right-side navigation.                  |
```

> | Content Manager     | Addition of the **open Gatsby preview** button in the right-side navigation                  |

Reviewer 2: ```suggestion
| Content Manager     | Addition of the **open Gatsby preview** button in the right-side navigation.                  |
```

Author: duplicate comment

There is no access to the GraphQL plugin in the admin panel. The GraphQL Playground is accessible at <http://localhost:1337/graphql> in a default Strapi application. More information on using the GraphQL API is located in the [API reference](/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md#graphql-api) and the [Developer Documentation plugins section](/developer-docs/latest/plugins/graphql.md#graphql).

Users can confirm the plugin installation by referencing the list of installed plugins by clicking on ![plugins icon](../assets/icons/plugins.svg) _Plugins_ in the main navigation.

### <img width="28" src="../assets/plugins/seo-logo.png"> SEO

The Strapi SEO plugin is designed to improve your application SEO. The Strapi SEO plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-seo). Once installed, the plugin is available in the main navigation and adds two pre-built components *MetaSocial* and *Seo* to the Content-type Builder.

To apply the SEO plugin to a Collection Type:

> 1. in the Content Builder select a Collection Type and click *Add another field*,

Author: With numbered lists, yes. I made the adjustment.

|Main navigation    | <ul> Addition of ![search icon](../assets/icons/search.svg) *SEO* to the main navigation. By clicking on ![search icon](../assets/icons/search.svg) *SEO* a list of Collection Types and Single Types, with the SEO status, is available.</ul>

### <img width="28" src="../assets/plugins/sentry.png"> Sentry

The Strapi Sentry plugin is used to track Strapi errors with Sentry. The Strapi Sentry plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-sentry). With the plugin installed a user can:

- initialize a Sentry instance when a Strapi application starts,
- send errors encountered in an application end API to Sentry,
- attach useful metadata to Sentry events, to help with debugging,
- expose a global Sentry service.

>  There is no access to the Sentry plugin in the admin panel. The user can confirm the plugin installation by referencing the list of installed plugins by clicking on ![plugins icon](../assets/icons/plugins.svg) *Plugins* in the main navigation.

Author: done
